modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/identity/governance/v1/dto/ConnectorsPatchReq.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Reply:
Fixed with https://github.com/wso2/product-is/pull/15931/commits/ae48c30d63e2a3955c1785ae5a4b2f1cd222d9dd